File: Payanak/Original/ClientModels/ChatModel.cs

using System.Collections.Generic;

namespace Backend.ClientModels{
    public class ChatModel
    {
        public string Avatar{get;set;}
        public string ChatClass{get;set;}
        public string ImagePath{get;set;}
        public string Time{get;set;}
        public List<string> Messages{get;set;}
        public string MessageType{get;set;}
        public long TicketId{get;set;}
    }
}

File: Payanak/Original/ClientApp/src/app/shared/model/chat.model.ts
export class ChatModel {
  public avatar: string;
  public chatClass: string;
  public imagePath: string;
  public time: string;
  public messages: string[];
  public messageType: string;
  public ticketId: number;

  public constructor(avatar: string, chatClass: string, imagePath: string, time: string, messages: string[], messageType: string) {
    this.avatar = avatar;
    this.chatClass = chatClass;
    this.imagePath = imagePath;
    this.time = time;
    this.messages = messages;
    this.messageType = messageType;
    this.ticketId = -1;
  }
}

